administrator/language/en-GB/en-GB.plg_behaviour_taggable.ini

```diff
@@ -0,0 +1,7 @@
+; Joomla! Project
+; Copyright (C) 2005 - 2015 Open Source Matters. All rights reserved.
+; License GNU General Public License version 2 or later; see LICENSE.txt, see LICENSE.php
+; Note : All ini files need to be saved as UTF-8
+
+PLG_BEHAVIOUR_TAGGABLE="Behaviour - Taggable"
+PLG_BEHAVIOUR_TAGGABLE_XML_DESCRIPTION="Allows content items to be tagged."
```

Comment:
Only database items with an attached content type can be tagged. Even under
the new content architecture which unifies the content building blocks
across all extensions it is still possible to have an item without a
content type, therefore impossible to tag (or version, the same applies
there). So I think "content items" used in this broader scope is more
accurate than "items".

*Nicholas K. Dionysopoulos*
*Director, Akeeba Ltd*
https://www.AkeebaBackup.com

On Sun, Aug 9, 2015 at 8:42 PM, Niels Braczek <notifications@github.com>
wrote:

> In administrator/language/en-GB/en-GB.plg_behaviour_taggable.ini
> <https://github.com/joomla-projects/joomla-pythagoras/pull/17#discussion_r36591122>
> :
>
> > @@ -0,0 +1,7 @@
> > +; Joomla! Project
> > +; Copyright (C) 2005 - 2015 Open Source Matters. All rights reserved.
> > +; License GNU General Public License version 2 or later; see LICENSE.txt, see LICENSE.php
> > +; Note : All ini files need to be saved as UTF-8
> > +
> > +PLG_BEHAVIOUR_TAGGABLE="Behaviour - Taggable"
> > +PLG_BEHAVIOUR_TAGGABLE_XML_DESCRIPTION="Allows content items to be tagged."
>
> In the Orthogonal Component Structure, this does not apply to content
> only. It should better read
> PLG_BEHAVIOUR_TAGGABLE_XML_DESCRIPTION="Allows items to be tagged."
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/joomla-projects/joomla-pythagoras/pull/17/files#r36591122>
> .
>